packages/forms/src/index.ts

```diff
@@ -212,6 +212,90 @@ export const addPatternToPage = (
   };
 };
 
+export const movePatternBetweenPages = (
+  bp: Blueprint,
+  sourcePageId: PatternId,
+  targetPageId: PatternId,
+  patternId: PatternId,
+  position: string,
+  isPageMove: boolean = false
+): Blueprint => {
+  const sourcePage = bp.patterns[sourcePageId] as PagePattern;
+  const targetPage = bp.patterns[targetPageId] as PagePattern;
+
+  if (sourcePage.type !== 'page' || targetPage.type !== 'page') {
+    throw new Error('Pattern is not a page.');
+  }
+
+  let updatedSourcePatterns;
+  let updatedTargetPatterns;
+
+  const sourcePagePatterns = sourcePage.data.patterns;
+  const indexToRemove = sourcePagePatterns.indexOf(patternId);
+  const newPattern = sourcePagePatterns.splice(indexToRemove, 1).toString();
+
+  if (isPageMove) {
+    // Moving a pattern within the source page
+    if (indexToRemove === -1) {
+      throw new Error('Pattern ID not found in the source page.');
+    }
+
+    updatedSourcePatterns = [
+      ...sourcePagePatterns.slice(0, indexToRemove),
+      ...sourcePagePatterns.slice(indexToRemove + 1),
+    ];
+
+    updatedTargetPatterns =
+      position === 'top'
+        ? [newPattern, ...updatedSourcePatterns]
+        : [...updatedSourcePatterns, newPattern];
+  } else {
+    // Moving a pattern between pages
+    if (indexToRemove === -1) {
+      throw new Error('Pattern ID not found in the source page.');
+    }
+
+    updatedSourcePatterns = [
+      ...sourcePagePatterns.slice(0, indexToRemove),
+      ...sourcePagePatterns.slice(indexToRemove + 1),
+    ];
+
+    updatedTargetPatterns =
+      position === 'top'
+        ? [patternId, ...targetPage.data.patterns]
+        : [...targetPage.data.patterns, patternId];
+  }
+
+  // console.log('patternId: ', patternId);
```

Comment:
Looks like this has some remnants from development/debugging. Can you remove the commented-out logging statements please?